Gemfile.lock

```diff
@@ -2,8 +2,8 @@ PATH
   remote: .
   specs:
     payrix (0.1.2)
-      faraday (~> 0.14)
-      faraday_middleware (~> 0.12)
```

Comment:
`faraday_middleware` is now [deprecated](https://github.com/lostisland/faraday_middleware#%EF%B8%8F-deprecation-warning-%EF%B8%8F) in favor of standalone middlewares (in separate gems). The specific one we are using is `faraday-follow_redirects` which is now being installed separately.